### PR TITLE
Darker background in in-active pane - Nemo

### PR DESCRIPTION
**Pull request by smurphos**
3rd attempt - ;-). Allows for a darker background in Nemo's inactive pane in dual pane view.

### CODE REVIEW

File: src/gtk-3.20/scss/apps/_nemo.scss [hunk 1]
```diff
@@ -17,6 +17,10 @@
     }
 
     .nemo-window {
+        .nemo-inactive-pane .view { 
+            background-color: $dark_bg_color;
```
Comment (actionless) [inline src/gtk-3.20/scss/apps/_nemo.scss]: shouldn't you also set `color: $dark-fg_color;`?

Comment (actionless) [reply src/gtk-3.20/scss/apps/_nemo.scss]: or, instead, use:

```scss
            background-color: mix($base_color, $text_color, .12);
```

Comment (smurphos) [reply src/gtk-3.20/scss/apps/_nemo.scss]: Thanks - sorry I didn't consider the color - I generally have the same for $fg_color and $dark_fg_color

The mix option is better though...

**Comment by actionless** [comment]
thanks!

**Pull request merged**